CSS/css_completions.py

```diff
@@ -426,7 +426,7 @@ class CSSCompletions(sublime_plugin.EventListener):
     regex = None
 
     def on_query_completions(self, view, prefix, locations):
-        selector_scope = "source.css - meta.selector.css"
+        selector_scope = "source.css - meta.selector.css, text.html meta.attribute-with-value.style.html string.quoted - punctuation.definition.string.begin.html" # match inside a CSS document and inside the style attribute of a HTML tag, including just before the quote that closes the attribute value
```

Comment:
Can you place the comment on the previous line and wrap the string to 120 chars?

I realize it isn't obvious with the current public/private repo split, but I recently updated all plugin code shipped in the default packages to pass flake8 with a max line-length of 120 chars.